### PR TITLE
Saas 12.3 search panel fixes dam

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

**Comment by aab-odoo** [comment]
robodoo rebase-ff

File: addons/web/models/models.py [hunk 1]
```diff
@@ -347,7 +347,7 @@ def group_id_name(value):
                     for group in groups
                 }
             # retrieve all possible values, and return them with their label and counter
-            selection = self.fields_get([field_name])[field_name]
+            selection = self.fields_get([field_name])[field_name]['selection']
```
Comment (aab-odoo) [inline addons/web/models/models.py:350]: missing test

File: addons/web/static/tests/views/search_panel_tests.js [hunk 2]
```diff
@@ -1611,6 +1611,51 @@ QUnit.module('Views', {
...
+                        '<field name="company_id" select="multi" groupby="char_field"/>' +
+                    '</searchpanel>' +
+                '</kanban>',
+        });
+
+        assert.containsN(kanban, '.o_search_panel_section', 1);
+        var $firstSection = kanban.$('.o_search_panel_section');
+
+        // There should be a group 'false' displayed with only value B inside it.
+        assert.containsN($firstSection, '.o_search_panel_filter_group', 1);
```
Comment (aab-odoo) [inline addons/web/static/tests/views/search_panel_tests.js:1646]: containsOnce

File: addons/web/models/models.py (hunk 1 above)
Comment (Polymorphe57) [reply addons/web/models/models.py:350]: no test here, the test module will appear in master

File: addons/web/static/tests/views/search_panel_tests.js (hunk 2 above)
Comment (Polymorphe57) [reply addons/web/static/tests/views/search_panel_tests.js]: done

**Comment by Polymorphe57** [comment]
> Second commit message: 'This commit avoids...'

done

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**